doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -147,6 +147,8 @@ Bluetooth samples
 
 * Added :ref:`peripheral_ams_client` sample.
 
+* Removed Peripheral Alexa Gadgets Bluetooth sample and Alexa Gadgets Service library due to Amazon pausing support for the Gadgets ecosystem.
```

Comment:
I believe the entry about the removal of the library needs to go under Libraries -> Bluetooth libraries and services, not combined to this entry.